modules/openapi-generator/src/main/resources/go-experimental/configuration.mustache

```diff
@@ -65,3 +114,28 @@ func NewConfiguration() *Configuration {
 func (c *Configuration) AddDefaultHeader(key string, value string) {
 	c.DefaultHeader[key] = value
 }
+
+// ServerUrl returns URL based on server settings
+func (c *Configuration) ServerUrl(index int, variables map[string]string) (string, error) {
+	server := c.Servers[index]
```

Comment:
Since this is an exported function that users actually could call in an arbitrary way, we should ensure that `index` is within length of `c.Servers` (and return error otherwise).

Reply:
Sure. Panic is not nice 👍